pulsar/internal/lookup_service_test.go

```diff
@@ -117,9 +117,10 @@ func TestLookupSuccess(t *testing.T) {
 
 		expectedRequests: []pb.CommandLookupTopic{
 			{
-				RequestId:     proto.Uint64(1),
-				Topic:         proto.String("my-topic"),
-				Authoritative: proto.Bool(false),
+				RequestId:              proto.Uint64(1),
+				Topic:                  proto.String("my-topic"),
+				Authoritative:          proto.Bool(false),
+				AdvertisedListenerName: proto.String(""),
```

Comment:
@xixiss Can you add new test case for this change? This test case is used to cover other scenarios, please do not verify our own logic in this test case.